codewof/users/forms.py

```diff
@@ -80,11 +80,61 @@ class UserChangeForm(forms.ModelForm):
         empty_label=None,
     )
 
+    remind_on_monday = forms.BooleanField(required=False, label='Monday')
+    remind_on_tuesday = forms.BooleanField(required=False, label='Tuesday')
+    remind_on_wednesday = forms.BooleanField(required=False, label='Wednesday')
+    remind_on_thursday = forms.BooleanField(required=False, label='Thursday')
+    remind_on_friday = forms.BooleanField(required=False, label='Friday')
+    remind_on_saturday = forms.BooleanField(required=False, label='Saturday')
+    remind_on_sunday = forms.BooleanField(required=False, label='Sunday')
+
+    timezone = forms.ChoiceField(
+        choices=User.TIMEZONES,
```

Comment:
This is a very long list, would recommend looking at using `common_timezones`.

> common_timezones is a list of useful, current timezones. It doesn’t contain deprecated zones or historical zones, except for a few I’ve deemed in common usage, such as US/Eastern (open a bug report if you think other timezones are deserving of being included here). It is also a sequence of strings.
> http://pytz.sourceforge.net/#helpers

Reply:
Using common_timezones reduces the list from 593 to 439. Still a pretty big list, but it is better. However, I've modified the timezones in the model instead of the form.